source/clear-linux/tutorials/wordpress/web-server-install.rst

```diff
@@ -1,15 +1,16 @@
 .. _web-server-install:
 
-Create a Clear Linux\* based web server
-#######################################
+Create a |CL-ATTR| based web server
```

Comment:
@krisdale. Check in with me on this title, in person, please. Example: `Create a LAMP stack server with |CL-ATTR|.` 
I recommend that we simplify these doc titles as part 1 and 2--or make them obviously sequential titles.